test/functional/test-experiments.js

```diff
@@ -43,11 +43,12 @@ describe('isExperimentOn', () => {
   });
 
   afterEach(() => {
-    resetExperimentToggles_();
+    resetExperimentTogglesForTesting();
```

Comment:
I'd kind of prefer we call this method in the `_init_tests.js`'s `afterEach`. Could you please at least try? It might cause too many errors, but I think it should be ok.